core/settings.py

```diff
@@ -186,9 +186,8 @@
 EMAIL_HOST = 'smtp.gmail.com'
 EMAIL_PORT = 587 # For TLS, 465 for SSL
 EMAIL_USE_TLS = True
-EMAIL_HOST_USER = 'deakinhardhatwebsite@gmail.com'
-EMAIL_HOST_PASSWORD = 'dtrv xywy rpqq tozw'
-
+EMAIL_HOST_USER = 'hardhatcompanywebsite@gmail.com'
+EMAIL_HOST_PASSWORD = 'nuje nbmo cfqe skjb'
```

Comment:
App password for email account should not be stored in git

Reply:
This is the code of the SMTP service, not the Password. The content displayed has no security implications. I also added additional content (including front-end and security aspects). Please check

Reply:
I'm still sure that storing an app access token in the repository is a massive security issue but we can handle that at a later date